### PR TITLE
support rails 3.0.9

### PR DESCRIPTION
**Pull request by skiz**
Works with rails 3.0.9 now (and should work with 3.1.0), I think the last version only worked up to like 3.0.2 or something. activemodel has changed quite a bit along with mass assignment among other oddities.

### CODE REVIEW

File: lib/active_presenter/version.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 module ActivePresenter
   module VERSION
     MAJOR    = 1
-    MINOR    = 3
+    MINOR    = 4
```
Comment (skiz) [inline lib/active_presenter/version.rb:4]: suitable?

File: lib/active_presenter/base.rb [hunk 2]
```diff
@@ -5,16 +5,16 @@ module ActivePresenter
     extend  ActiveModel::Callbacks
     extend  ActiveModel::Naming
     extend  ActiveModel::Translation
+    include ActiveModel::MassAssignmentSecurity
     include ActiveModel::Conversion
 
     attr_reader :errors
-    
+
```
Comment (skiz) [inline lib/active_presenter/base.rb:12]: lots of whitespace corrections in this request.

**Comment by mguterl** [comment]
@skiz - I took a look at your Rails3 branch while upgrading one of our projects.  It seems to clean up the majority of the warnings, however, we are still getting one warning about class_inheritable_attribute.

DEPRECATION WARNING: class_inheritable_attribute is deprecated, please use class_attribute method instead. Notice their behavior are slightly different, so refer to class_attribute documentation first. (called from /Users/ry/.rvm/gems/ree-1.8.7-2011.03@midwife/bundler/gems/active_presenter-d0aa21943aa6/lib/active_presenter/base.rb:15)

**Comment by skiz** [comment]
@mguterl - I initially updated this gem to support rails 3.0.x post beta as the current one is completely broken and only works for 3.0.0-rcX iirc.  I haven't encountered this issue and am using this branch in production currently.  Do you have some context as to where this is happening? I will take a look and make the appropriate changes as needed.

**Comment by mguterl** [comment]
@skiz - the deprecation warning shows up for us when we run the specs in our rails project on Rails 3.1.

**Comment by andyl** [comment]
I'm getting the deprecation warning using Rails 3.1.0 and active_presenter 1.4.0

**Pull request merged**